images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -161,6 +161,75 @@ Hashtable for service arguments
     }
 }
 
+function Install-VsixExtension
+{
+    Param
+    (
+        [String]$Url,
+        [String]$Name
+    )
+
+    $FilePath = "${env:Temp}\$Name"
+    $exitCode = -1
```

Comment:
This variable is always replaced on 217 line and we don't need it. Or we should change logic and put the variable in try-catch block 208-217,

Reply:
It's also a part of the old function. I guess they didn't know that empty != 0 😃
Removed.